staff-finding/scripts/train_staffline_detector.py

```
@@ -0,0 +1,278 @@
+def remap_label_file(src: Path, dst: Path) -> None:
+    """Copy a YOLO .txt label, remapping SOURCE_CLASS → TARGET_CLASS."""
+    lines: list[str] = []
+    with src.open() as fh:
+        for raw in fh:
+            parts = raw.strip().split()
+            if not parts:
+                continue
+            cls = int(parts[0])
+            if cls == SOURCE_CLASS:
+                cls = TARGET_CLASS
+            lines.append(" ".join([str(cls)] + parts[1:]))
```

Ruff (0.15.21)

[warning] Consider `[str(cls), *parts[1:]]` instead of concatenation

Replace with `[str(cls), *parts[1:]]`

(RUF005)